sensors/video/sensorhub-driver-rtpcam/build.gradle

```diff
@@ -23,6 +23,14 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
     attributes('Bundle-Activator': 'org.sensorhub.impl.sensor.rtpcam.Activator')
+    attributes('-exportcontents': 'org.sensorhub.impl.sensor.rtpcam')
+    attributes('Import-Package': '''
```

Comment:
Since jcodec is only used in tests, perhaps replacing testCompile by testImplementation above would work better than excluding all these packages

Reply:
Good point: 38dcac5da338c129e1228e18b1581381187a51ab